ui/lib/kv/addon/utils/kv-breadcrumbs.js

```diff
@@ -32,8 +32,8 @@ function splitSegments(secretPath) {
  * @param {boolean} lastItemCurrent
  * @returns array of breadcrumbs specific to KV engine
  */
-export function breadcrumbsForSecret(secretPath, lastItemCurrent = false) {
-  if (!secretPath) return [];
+export function breadcrumbsForSecret(backend, secretPath, lastItemCurrent = false) {
```

Comment:
It'd be helpful if these args were named args, but I understand that's likely outside the scope of this PR